pkg/fanal/image/registry/google/google_test.go

```diff
@@ -23,21 +21,18 @@ func TestCheckOptions(t *testing.T) {
 		},
 		"NoOption": {
 			domain: "gcr.io",
-			gcr:    &Registry{domain: "gcr.io"},
+			gcr:    &Registry{},
 		},
 		"CredOption": {
 			domain: "gcr.io",
 			opt:    types.RegistryOptions{GCPCredPath: "/path/to/file.json"},
-			gcr: &Registry{
```

Comment:
IIUC we should compare `GoogleRegistryClient`.

Reply:
@DmitriyLewen Should be good now, thanks for catching this